target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -533,6 +534,10 @@ ath79_setup_macs()
 		lan_mac=$(macaddr_setbit $base_mac 29)
 		[ $lan_mac = $base_mac ] && lan_mac=$(macaddr_unsetbit $base_mac 29)
 		;;
+	asus,rp-ac66)
+		lan_mac=$(mtd_get_mac_binary art 0x1002)
```

Comment:
@therealsummoner 

This is redundant and should be removed, since it's already in the DTS.